src/routes/basic-form.tsx

```diff
@@ -0,0 +1,255 @@
+              dispatch({type: "email", payload: { email: value }}) 
+            } 
+          />    
+        </GoAFormItem>
+
+        <GoAFormItem label="Phone number input" error={errors.phone}>
+          <GoAInputTel
+            width="100%" 
+            name="phone" 
+            value={state.phone} 
+            error={!!errors.phone}
+            onChange={(_, value) => 
+              dispatch({type: "phone", payload: {phone: value}}) 
+            } 
+          />    
+        </GoAFormItem>
+      </GoAFlexRow>
+
+      <GoAFlexRow gap="medium">
+        <GoAFormItem label="Text area" error={errors.textarea} helpText="The text area can count the number of characters a user inputs.">
+          <GoATextArea 
+            width="100%" 
+            name="textarea" 
+            value={state.textarea} 
+            error={!!errors.textarea}
+            showCounter={true} 
+            maxCharCount={200}
+            onChange={(_, value) => 
+              dispatch({ type: "textarea", payload: {"textarea": value} }) 
+            } 
+          />
+        </GoAFormItem>
+      </GoAFlexRow>
+
+      <GoAFlexRow gap="medium">
+        <GoAFormItem label="Do you want to show another type of input?">
+          <GoARadioGroup 
+            name="moreInput"
+            value={state.moreInput ? "yes" : "no"}
+            onChange={(_, value) => dispatch({ type: "moreInput", payload: {"moreInput": value === "yes"} }) } 
+          >
+            <GoARadioItem key="yes" name="moreInput" value="yes" label="Yes" />
+            <GoARadioItem key="no" name="moreInput" value="no" label="No" />
+          </GoARadioGroup>
+        </GoAFormItem>
+      </GoAFlexRow>
+
+      {state.moreInput &&
+        <GoAContainer type="interactive" headingSize="large">
+          <div style={{display: "flex", justifyContent: "space-between", alignItems: "baseline"}}>
+            <h3>This is an interactive container</h3>
+            <GoAButton onClick={showSaveConfirmationModal} type="tertiary">Remove container</GoAButton>
+          </div>
+          <p>
+            You can use a container to group related content and tasks. An interactive container indicates that there is content within actionable, and is denoted by the teal colour band.
+          </p>
+
+          <GoAFormItem 
+            label="Do you want to show another type of input?"
+            helpText="You can select multiple items on this list."
+          >
+            {Items.map(item => 
+              <GoACheckbox 
+                key={item}
+                name="items"
+                value={item}
+                checked={state.items.includes(item)}
+                onChange={(_, checked, value) => {
+                  dispatch({type: "items", payload: { item: value, checked }})
+                }}
+                text={item}
+              />
+            )}
+          </GoAFormItem>
+        </GoAContainer>
+      }
+
+      <div className="space-3"></div>
```

Comment:
Is this what we're using for margin between elements?

Reply:
for right now